packages/mobile/src/screens/edit-track-screen/fields/CoverAttributionField.tsx

```diff
@@ -0,0 +1,28 @@
+import { useField } from 'formik'
+
+import type { ContextualMenuProps } from 'app/components/core'
+import { ContextualMenu } from 'app/components/core'
+
+import { IS_COVER } from '../screens/CoverAttributionScreen'
+
+const messages = {
+  coverAttribution: 'Cover Attribution',
+  isCover: 'Cover'
```

Comment:
super nit but can we rename this message to `cover` so it doesn't collide with the `isCover` boolean?